Test/CheckFilterTest.cs

```diff
@@ -18,5 +20,41 @@ public void TestFilters()
             Assert.IsTrue(LibPcapLiveDevice.CheckFilter("port 23", out errorString));
             Assert.IsNull(errorString);
         }
+
+        /// <summary>
+        /// Test RunBpfProgram() and other filter related methods
+        /// </summary>
+        [Test]
+        public void FilterMethods()
+        {
+            using var device = TestHelper.GetPcapDevice();
```

Comment:
@chmorgan this adapter is not necessarily the internet adapter, thus `WebFetch` may not have an effect on it.

Reply:
This same approach does work for the WinDivert test though. That's why I figured it would be ok here.

Reply:
Windivert is a layer 3 driver, libpcap is layer 2. a reliable solution would be to use the ARP class for example.